tests/integration-tests/tests/common/data/system-analyzer.sh

```diff
@@ -167,7 +167,7 @@ function main() {
   local TEMP_DIR="${BASE_TEMP_DIR}/system-information/"
 
   # Register signal handling to clean the temporary directory in case of kill, kill -9, ctrl+c, error in the script
-  trap 'signal_handler ${?} ${LINENO} ${BASE_TEMP_DIR}' SIGKILL SIGINT SIGTERM SIGHUP INT ERR EXIT
+  trap 'signal_handler ${?} ${LINENO} ${BASE_TEMP_DIR}' SIGINT SIGTERM SIGHUP INT ERR EXIT
```

Comment:
The next time we're fixing ShellCheck findings is worth adding in the PR description the links of the findings. E.g. https://github.com/koalaman/shellcheck/wiki/SC2173

This will help the reviewer to understand the change and share the best practice.